### ISSUE TITLE
Popover focus shifts to 'dismiss' button before user can read the component's content

### ISSUE THREAD

**Issue by paulkenney** [adobe/react-spectrum#880]
# 🐛 Bug Report 

I have an issue with how VoiceOver/Safari is interacting with react-spectrum v3 popovers. Focus seems to automatically/quickly move from the content of the popover to the “dismiss” button, and not much time is given to the user to process the popover content.

## 🤔 Expected Behavior

When the Popover is rendered, the screen reader should announce the Popover's content and not place focus on elements outside of that content.

## 😯 Current Behavior

Using OSX/Safari/VoiceOver:

When the Popover is opened, (visible) focus is moved to the hidden 'dismiss' button instead of the content.

![image](https://user-images.githubusercontent.com/2644630/89214227-4a273a80-d57b-11ea-8748-9f0ed8659da0.png)

## 💁 Possible Solution
NA

## 🔦 Context
This is an accessibility issue.

## 💻 Code Sample
NA

## 🌍 Your Environment

| Software         | Version(s) |
| ---------------- | ---------- |
| react-spectrum   | 3.x
| Browser          | Safari
| Operating System | OSX
| Screen Reader | VoiceOver

## 🧢 Your Company/Team
Adobe/AEP

## 🕷 Tracking Issue (optional)
NA

**Comment by devongovett** [comment]
Is your app inside an iframe? If so, this is a Safari bug where it autofocuses the first focusable element inside an `aria-modal` container. See https://bugs.webkit.org/show_bug.cgi?id=211934

**Comment by paulkenney** [comment]
Yes, it does run inside an iframe.
